src/classes/AccountDeDuper.cls

```diff
@@ -1,4 +1,6 @@
-public with sharing class AccountDeDuper {
+public with sharing class Account_De_Duper {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.